### PR TITLE
web: release info for 2023.4

### PR DESCRIPTION
**Pull request by ajstrongdev**
- [x] Update download mirrors
- [x] Update download page information
- [x] Write release notes
- [x] Modify news page
- [x] Add banner across homepage

### CODE REVIEW

**Comment by ajstrongdev** [comment]
Release notes are finished

**Comment by oklopfer** [comment]
Add to release notes:
- rhino-core/rhino-pine-core/rhino-ubxi-core/rhino-server-core are now set as essential packages
- rhino-system light mode bug fixed
- connecting to wifi on embedded fixed
- pacstall-qa introduced as a new default package
- rhino-setup is now properly on the system at post-install
- kernels 6.6.7-generic, 6.7.0-rc5-okpine, 6.5.0-raspi

File: news-10.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    The "Your System" application now displays the version number as a pill, and can be copied to the clipboard on click.
+  </p>
+  <h1 class="text-off-white text-center text-3xl mt-4">
+    What to expect in 2024
+  </h1>
+  <p class="text-off-white mt-4">
+    One of the larger things to expect in 2024, is the debut of our new icon pack! We've been hard at work designing an icon pack that is both visualy appealing whilst keeping the essence of the original artwork.
+  </p>
+  <p class="text-off-white mt-4">
+    We also plan to create a Linux kernel management application. This application will allow for you to change between different versions of the linux kernel, an example of this could be switching to the long-term support kernel for more stability.
```
Comment (oklopfer) [inline news-10.html:118]: no we dont, this is called mainline and it exists.

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: news-10.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  </p>
+  <p class="text-off-white mt-4">
+    Some other things to expect in 2024 would be support for offline installations, as well as improved documentation.
```
Comment (oklopfer) [inline news-10.html:121]: - offline installions
- updated wiki pages
- prospects of wayland
- ubxi ports
- rhino-pkg rewrite in nushell
- pacstall 5.0.0, which will bring significantly closer pacscript parity to PKGBUILDs
most of these can be broken out into their own bullet points, too.

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: news-10.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; The PineTab2 now features a more user-friendly experimental WiFi module enablement.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; All PinePhones and PineTabs now offer flashlight support.
+  </p>
+  <p class="text-off-white mt-4">
+    All of these changes were brought over directly from Oren Klopfer's Ubuntu Touch port.
+  </p>
```
Comment (oklopfer) [inline news-10.html:98]: Add to release notes:
- rhino-core/rhino-pine-core/rhino-ubxi-core/rhino-server-core are now set as essential packages
- rhino-system light mode bug fixed
- connecting to wifi on embedded fixed
- pacstall-qa introduced as a new default package
- rhino-setup is now properly on the system at post-install
- kernels 6.6.7-generic, 6.7.0-rc5-okpine, 6.5.0-raspi

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: news-10.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        <h2 class="text-off-white text-2xl font-light mt-2 drop-shadow-header-drop">
+          Find news relating to Rhino Linux & its development
+        </h2>
+        <br>
+    </div>
+</div>
+
+<div class="container w-[95vw] lg:w-[60vw] mt-4 rounded-[0.65em] p-4 m-auto shadow-2xl bg-site-300">
+  <h1 class="text-center text-off-white text-3xl">
+    Ending the year - Rhino Linux 2023.4
```
Comment (oklopfer) [inline news-10.html:59]: Rhino Linux 2023.4 - End of year wrap-up and major stability improvements

File: news.html [hunk 5]
```diff
@@ -59,9 +59,22 @@ <h2 class="text-off-white text-2xl font-light mt-2 drop-shadow-header-drop">
 
             <h1 class="text-off-white text-3xl text-center" id="news">News</h1>
 
-            <div class="mt-4 container m-auto">
+            <div class="mt-4 container m-auto lg:grid lg:grid-cols-2">
+
+                <div class="bg-site-200 rounded-[0.65em] text-off-white p-4 shadow-2xl mt-4 lg:mr-2">
+                    <h1 class="text-3xl text-off-white">
+                        Rhino Linux 2023.4: Enhancing the experience
```
Comment (oklopfer) [inline news.html:66]: Experience

File: news-10.html (hunk 3 above)
Comment (oklopfer) [reply news-10.html:98]: explain what pacstall-qa is too

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: news-10.html (hunk 1 above)
Comment (ajstrongdev) [reply news-10.html:118]: Hm, that app doesn't have functionality with the pacscripts we install, perhaps we could create a fork of it instead. + I'd want it done in libadwaita if we're shipping it with the distribution anyway.

File: news-10.html (hunk 2 above)
Comment (ajstrongdev) [reply news-10.html:121]: Got it, will fix

File: news-10.html [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+    &bull; We have managed to close an open issue, where bluetooth support would be hit-and-miss on certain devices, this has now been resolved.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; A bug with our auto-tiling implementation (utilising cortile) and conky has been resolved.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; The "Your System" application now displays the version number as a pill, and can be copied to the clipboard on click.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; Pacstall 4.3.0 released, meaning that packages can now be tagged as "essential", and as such cannot be removed from the system. The rhino-core package, and its varients are now tagged as essential, meaning they’re assumed unremovable once installed. This will also mean unicorn is not automatically removed when switching between -cores, this must be a deliberate action. This makes Rhino Linux more stable in the long-term. If you wish to remove the Unicorn Desktop environment, you first must install rhino-server-core or rhino-ubxi-core, remove unicorn-desktop, and then install your preferred environment.
```
Comment (oklopfer) [inline news-10.html:112]: this needs to be worked on. I will look over it tomorrow

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: news-10.html (hunk 6 above)
Comment (ajstrongdev) [reply news-10.html:112]: In what way?

Comment (oklopfer) [reply news-10.html:112]: Beginning with Pacstall 4.3.0, packages may now use Debian’s `priority` flag. As such, `rhino-core` and variants are now tagged `essential`, marking them unremovable to the system. Additionally, Unicorn will no longer automatically be removed when switching between the `rhino*-core` family. To remove `unicorn-desktop` or `unicorn-mobile`, users must first switch to `rhino-ubxi-core` or `rhino-server-core`; they will then be able to safely remove the package. This step is also needed if one is switching between `rhino-core` and `rhino-pine-core`.

File: news-10.html [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+<div class="container w-[95vw] lg:w-[60vw] mt-4 rounded-[0.65em] p-4 m-auto shadow-2xl bg-site-300">
+  <h1 class="text-center text-off-white text-3xl">
+    Rhino Linux 2023.4 - Enhancing the Experience, and looking towards 2024
+  </h1>
+  <p class="text-off-white mt-4">
+    After a long development period we're ready to finish off the year with a stellar release, as well as inform you, the community, of what to expect in 2024.
+  </p>
+  <p class="text-off-white mt-4">
+    This development period was substantially longer than our previous releases, this can be attributed to the start of a new semester. If you wish to contribute to Rhino Linux and help us push out amazing upgrades faster, consider joining either our <a href="contact.html" class="text-rhino-purple">Matrix or Discord</a>.
```
Comment (oklopfer) [inline news-10.html:65]: which can be attributed to

File: news-10.html [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+    What to expect in 2024
+  </h1>
+  <p class="text-off-white mt-4">
+    One of the larger things to expect in 2024, is the debut of our new icon pack! We've been hard at work designing an icon pack that is both visualy appealing whilst keeping the essence of the original artwork.
+  </p>
+  <p class="text-off-white mt-4">
+    As we mentioned before about the Unicorn Beyond XFCE Initiative, in 2024 we will be aiding the development and creation of UBXI ports.
+  </p>
+  <p class="text-off-white mt-4">
+    Rhino-pkg will be getting a full overhaul in nushell.
```
Comment (oklopfer) [inline news-10.html:140]: `rhino-pkg` in lowercase + code style

Also, let's rephrase:

`rhino-pkg` is in the process of being rewritten in nushell. This change will be a complete overhaul of the program, aiming to make it faster, more accurate, more precise, and introducing several new functions, like the ability to sync repositories without upgrading, display descriptions in search results, and more.

File: news-10.html [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+    &bull; <a href="https://github.com/rhino-linux/unicorn/issues/39" class="text-rhino-purple">Unicorn #39</a> has finally been merged, bringing optional auto-tiling to the Unicorn Desktop. Kick start your tiling window experience now, on start up you will notice an applet in your panel representing whether you are using our floating or tiling window mode.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; uLauncher now appears more rounded, and has a slightly different background colour.
+  </p>
+  <h2 class="text-off-white text-2xl mt-4">
+    rhino-pkg
+  </h2>
+  <p class="text-off-white mt-4">
+    &bull; Changes in Nala v0.14.0 have affected how upgrades are handled. As a result, rhino-pkg now passes <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">sudo nala upgrade --full --no-autoremove</code> as the default upgrade functionality, providing Nala is installed and enabled.
```
Comment (oklopfer) [inline news-10.html:82]: No. Just keep it simple, and add a second bullet:

- If Nala is installed, `rhino-pkg update` no longer auto-removes packages by default. 

- To clean up unneeded packages or broken dependencies on a system, users can run the new `rhino-pkg cleanup` command. 


Also, move the 4.3.0/priority thing up to this (the rhino-pkg) section.

File: news-10.html [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+    &bull; The PineTab2 now features a more user-friendly experimental WiFi module enablement.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; All PinePhones and PineTabs now offer flashlight support.
+  </p>
+  <p class="text-off-white mt-4">
+    All of these changes were brought over directly from Oren Klopfer's Ubuntu Touch port.
+  </p>
+  <h2 class="text-off-white mt-4 text-2xl">
+    Other changes
```
Comment (oklopfer) [inline news-10.html:100]: Split up Features and Bug Fixes.

File: news-10.html [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+  <p class="text-off-white mt-4">
+    This development period was substantially longer than our previous releases, this can be attributed to the start of a new semester. If you wish to contribute to Rhino Linux and help us push out amazing upgrades faster, consider joining either our <a href="contact.html" class="text-rhino-purple">Matrix or Discord</a>.
+  </p>
+  <h1 class="text-off-white text-3xl text-center mt-4">
+    Release notes:
+  </h1>
+  <p class="text-off-white mt-4">To upgrade to 2023.4 please run: <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">rpk update -y</code>.</p>
+  <h2 class="text-2xl text-off-white mt-4">The Unicorn Desktop</h2>
+  <p class="text-off-white mt-4">
+    &bull; <a href="https://github.com/rhino-linux/unicorn/issues/39" class="text-rhino-purple">Unicorn #39</a> has finally been merged, bringing optional auto-tiling to the Unicorn Desktop. Kick start your tiling window experience now, on start up you will notice an applet in your panel representing whether you are using our floating or tiling window mode.
```
Comment (oklopfer) [inline news-10.html:73]: Unicorn #39 has finally been merged, bringing optional auto-tiling to Unicorn. After updating and rebooting, a new applet will appear in the top right section of the panel. To kick start your tiling window experience, you can select the new applet, which will bring up a toggle to enable/disable tiling mode.

File: news-10.html [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+    rhino-pkg
+  </h2>
+  <p class="text-off-white mt-4">
+    &bull; Changes in Nala v0.14.0 have affected how upgrades are handled. As a result, rhino-pkg now passes <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">sudo nala upgrade --full --no-autoremove</code> as the default upgrade functionality, providing Nala is installed and enabled.
+  </p>
+  <h2 class="text-off-white mt-4 text-2xl">
+    Pine64
+  </h2>
+  <p class="text-off-white mt-4">
+    &bull; Modem stability fixes have been shipped to both the PinePhone and the PinePhone Pro, as well as GPS support enablement
```
Comment (oklopfer) [inline news-10.html:88]: end this with a period

File: news-10.html [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+    &bull; All PinePhones and PineTabs now offer flashlight support.
+  </p>
+  <p class="text-off-white mt-4">
+    All of these changes were brought over directly from Oren Klopfer's Ubuntu Touch port.
+  </p>
+  <h2 class="text-off-white mt-4 text-2xl">
+    Other changes
+  </h2>
+  <p class="text-off-white mt-4">
+    &bull; We have managed to close an open issue, where bluetooth support would be hit-and-miss on certain devices, this has now been resolved.
```
Comment (oklopfer) [inline news-10.html:103]: No,: 

An issue regarding Bluetooth support has been resolved, with our full transition from `pulseaudio` to `pipewire`.

File: news-10.html [hunk 14]
```diff
@@ -0,0 +1,166 @@
...
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; A bug with our auto-tiling implementation (utilising cortile) and conky has been resolved.
```
Comment (oklopfer) [inline news-10.html:106]: Nope. conky was a user installed package. We did not fix anything. They fixed their own issue by editing their conky config. Remove this entirely, we did nothing.

File: news-10.html [hunk 15]
```diff
@@ -0,0 +1,166 @@
...
+    
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; pacstall-qa is now introduced as a new default package. pacstall-qa is a tool developed by Pacstall as a tool to test pascripts from PR's locally. You can read more about pacstall-qa <a href="https://github.com/pacstall/pacstall-qa" class="text-rhino-purple">here</a>.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; During the 2023.4 development cycle we announced the launch of the <a href="news-9.html" class="text-rhino-purple">The Unicorn Beyond XFCE initiative</a> as a way to create official ports of Unicorn to other desktop environments.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; The installed kernels for this release are 6.6.7-generic for desktop, 6.7.0-rc5-okpine on Pine64 devices and 6.5.0-raspi for Raspberry Pi.
```
Comment (oklopfer) [inline news-10.html:122]: Make the versions in code style

File: news-10.html [hunk 16]
```diff
@@ -0,0 +1,166 @@
...
+    One of the larger things to expect in 2024, is the debut of our new icon pack! We've been hard at work designing an icon pack that is both visualy appealing whilst keeping the essence of the original artwork.
+  </p>
+  <p class="text-off-white mt-4">
+    As we mentioned before about the Unicorn Beyond XFCE Initiative, in 2024 we will be aiding the development and creation of UBXI ports.
+  </p>
+  <p class="text-off-white mt-4">
+    Rhino-pkg will be getting a full overhaul in nushell.
+  </p>
+  <p class="text-off-white mt-4">
+    Pacstall 5.0.0 should release in 2024, which will bring significantly closer pacscript parity to PKGBUILDs.
```
Comment (oklopfer) [inline news-10.html:143]: Pacstall 5.0.0 *will* be releasing in 2024,

File: news-10.html [hunk 17]
```diff
@@ -0,0 +1,166 @@
...
+  </p>
+  <p class="text-off-white mt-4">
+    There are also prospects of Unicorn becoming available on Wayland in 2024.
```
Comment (oklopfer) [inline news-10.html:146]: Nope. Here: 

We are still exploring the prospects of Unicorn switching to Wayland in the future. Progress on `xfwm4-wayland` has not moved for the majority of our development this year, and so we are considering alternative compositing options. UBXI projects may help largely with this process, so we encourage you to contribute!

File: news-10.html [hunk 18]
```diff
@@ -0,0 +1,166 @@
...
+  </p>
+  <p class="text-off-white mt-4">
+    Some other things to expect in 2024 would be support for offline installations, as well as improved documentation on our <a href="wiki.html" class="text-rhino-purple">Rhino Linux wiki</a>.
```
Comment (oklopfer) [inline news-10.html:149]: ...to expect in 2024 *are* support for offline installations...

not "would be"

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: news-10.html [hunk 19]
```diff
@@ -0,0 +1,165 @@
...
+    &bull; <a href="https://github.com/rhino-linux/unicorn/issues/39" class="text-rhino-purple">Unicorn #39</a> has finally been merged, bringing optional auto-tiling to Unicorn. After updating and rebooting, a new applet will appear in the top right section of the panel. To kick start your tiling window experience, you can select the new applet, which will bring up a toggle to enable/disable tiling mode.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; uLauncher now appears more rounded, and has a slightly different background colour.
+  </p>
+  <h2 class="text-off-white text-2xl mt-4">
+    rhino-pkg
+  </h2>
+  <p class="text-off-white mt-4">
+    &bull; If Nala is installed, <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">rhino-pkg update</code> update no longer auto-removes packages by default. <br> &bull; To clean up unneeded packages or broken dependencies on a system, users can run the new <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">rhino-pkg cleanup</code> command.
```
Comment (oklopfer) [inline news-10.html:82]: rhino-pkg update update 😐

Also these should be two separate paragraph blocks if they're two separate bullets, just like everyone else.

File: news-10.html [hunk 20]
```diff
@@ -0,0 +1,165 @@
...
+    &bull; During the 2023.4 development cycle we announced the launch of the <a href="news-9.html" class="text-rhino-purple">The Unicorn Beyond XFCE initiative</a> as a way to create official ports of Unicorn to other desktop environments.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; The installed kernels for this release are <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">6.6.7-generic</code> for desktop, <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">6.7.0-rc5-okpine</code> on Pine64 devices and <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">6.5.0-raspi</code> for Raspberry Pi. 
+  </p>
+  <h2 class="text-off-white mt-4 text-2xl">
+    Bug fixes
+  </h2>
+  <p class="text-off-white mt-4">
+    &bull; An issue regarding Bluetooth support has been resolved, with our full transition from pulseaudio to pipewire.
```
Comment (oklopfer) [inline news-10.html:121]: pulse and pipe should be in code style.

File: news-10.html [hunk 21]
```diff
@@ -0,0 +1,165 @@
...
+    All of these changes were brought over directly from Oren Klopfer's Ubuntu Touch port.
+  </p>
+  <h2 class="text-off-white mt-4 text-2xl">
+    Other new features
+  </h2>
+  <p class="text-off-white mt-4">
+    &bull; The "Your System" application now displays the version number as a pill, and can be copied to the clipboard on click.
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; pacstall-qa is now introduced as a new default package. pacstall-qa is a tool developed by Pacstall as a tool to test pascripts from PR's locally. You can read more about pacstall-qa <a href="https://github.com/pacstall/pacstall-qa" class="text-rhino-purple">here</a>.
```
Comment (oklopfer) [inline news-10.html:109]: put pacstall-qa in code style.

File: news-10.html [hunk 22]
```diff
@@ -0,0 +1,165 @@
...
+    &bull; The installed kernels for this release are <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">6.6.7-generic</code> for desktop, <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">6.7.0-rc5-okpine</code> on Pine64 devices and <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">6.5.0-raspi</code> for Raspberry Pi. 
+  </p>
+  <h2 class="text-off-white mt-4 text-2xl">
+    Bug fixes
+  </h2>
+  <p class="text-off-white mt-4">
+    &bull; An issue regarding Bluetooth support has been resolved, with our full transition from pulseaudio to pipewire.
+  </p>``
+  <p class="text-off-white mt-4">
+    &bull; The "Your System" application only displaying in light mode bug has been fixed.
```
Comment (oklopfer) [inline news-10.html:124]: The bug causing the "Your System" application to be stuck in light mode has finally been resolved.

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: news-10.html [hunk 23]
```diff
@@ -0,0 +1,165 @@
...
+  </p>
+  <p class="text-off-white mt-4">
+    &bull; The installed kernels for this release are <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">6.6.7-generic</code> for desktop, <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">6.7.0-rc5-okpine</code> on Pine64 devices and <code class="text-off-white bg-site-200 rounded-[0.65em] p-1 border-rhino-purple border-2">6.5.0-raspi</code> for Raspberry Pi. 
+  </p>
+  <h2 class="text-off-white mt-4 text-2xl">
+    Bug fixes
+  </h2>
+  <p class="text-off-white mt-4">
+    &bull; An issue regarding Bluetooth support has been resolved, with our full transition from pulseaudio to pipewire.
+  </p>``
```
Comment (oklopfer) [inline news-10.html:122]: Stray ``

**Review by oklopfer — CHANGES_REQUESTED** [review]

**Pull request merged**